src/models/hooks/activityReport.js

```diff
@@ -23,6 +23,9 @@ const {
   removeCollaboratorsForType,
 } = require('../helpers/genericCollaborator');
 const { destroyLinkedSimilarityGroups } = require('./activityReportGoal');
+const { purifyFields } = require('../helpers/purifyFields');
+
+const AR_FIELDS_TO_ESCAPE = ['additionalNotes', 'context'];
```

Comment:
Should we also apply this to Objective TTA Provided as its also rich text (respectively)?

Reply:
I think we should start with the fields flagged by the pen test

Reply:
Ok lets create a ticket for it so its not lost.